java/nashorn.execution/manifest.mf

```diff
@@ -4,4 +4,4 @@ Comment: OpenIDE-Module-Layer: org/netbeans/modules/nashorn/execution/resources/
 OpenIDE-Module-Localizing-Bundle: org/netbeans/modules/nashorn/execution/Bundle.properties
 OpenIDE-Module-Specification-Version: 1.16
 AutoUpdate-Show-In-Client: false
-
+OpenIDE-Module-Recommends: org.netbeans.libs.graaljs
```

Comment:
Suspicious at first sight. I'd expect `cnb.org.netbeans.libs.graaljs` as that is the token exported by default by the module system for the `org.netbeans.libs.graaljs` module. However correct, the module explicitly exports `org.netbeans.libs.graaljs` token as can be seen below.

Reply:
`cnb.` could be better... it actually depends on the GraalJS library being defined rather than code from the module. Do you think it should rather depend on the module as such (rather than on the 'library' feature "token") ?

Reply:
As you wish.